bin/server.ts

```diff
@@ -12,6 +12,7 @@ import '../src/Controller/v1/ItemsController'
 import '../src/Controller/v1/PaymentsController'
 import '../src/Controller/v1/WebSocketsController'
 import '../src/Controller/v2/PaymentsControllerV2'
+import '../src/Controller/v2/UsersControllerV2'
```

Comment:
@karolsojko not sure if this should go to V2?

Reply:
@karolsojko bump

Reply:
@antsgar should be `v1` ... the `/subscriptions` endpoint has a `v2` because we already have that one at `v1` where it returns a different structure of data